.github/workflows/release.yaml

```diff
@@ -0,0 +1,15 @@
+name: release
+on:
+  push:
+    branches:
+      - main
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
For [UW-317](https://jira.epic.oarcloud.noaa.gov/browse/UW-317) work, steps to build the `uwtools` package and publish it to anaconda.org will follow this step. Tags will be create after the package is staged and ready to use.